app-mobilephone/gammu/gammu-1.38.5.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
I'm lost here. Why are you changing this line if you didn't modify the file?

Also this commit doesn't add anything, so please fix the commit message. Also please fix spacing between commit message lines according to GLEP66. Also you can put versions straight into summary line.